int32[] ids
VehicleTraj[] surround_traj
